uptane-standard.md

```diff
@@ -880,23 +880,22 @@ If {{TAP-5}} is supported, a primary ECU SHALL download metadata and images foll
-6. For each image name for which the previous two steps **stored** image metadata, obtain verified image metadata for that image by iterating over each mapping in the repository mapping metadata, in order, and performing the following steps for each mapping:
-    1. If the image name does not match the image paths listed in the mapping, skip to the next mapping.
-    2. For each repository listed in this mapping, if top-level metadata has not been refreshed for that repository using the process in {{refresh_toplevel_metadata}} in this iteration of the {{full_verification}} procedure, refresh top-level metadata for that repository using the steps in {{refresh_toplevel_metadata}}.
-    3. For each repository listed in this mapping, perform the procedure in {{resolve_delegations}}, beginning at the top-level Targets metadata in that repository, to find metadata for the image if it exists, and **store** it if it does.
-    4. Find the first set of at least X repositories -- where X is the mapping's threshold, and such that the Director repository is in the set -- for which all of the following is true for the stored image metadata for each repository:
-        1. The non-custom Targets metadata (i.e., length and hashes) of the unencrypted image are the same.
-        2. If listed, the "MUST match" custom Targets metadata (e.g., hardware identifier and release counter) are the same.
-        3. The ECU Identifier is the same across repositories, for all repositories that list it.
-    5. If there is no such set (of at least the mapping's threshold number of repositories meeting the criteria in the prior step), then check the terminating property on this mapping.  If the terminating value for this mapping is true, then abort the search for image metadata for this image, discarding metadata **stored** for this image, as there is no verifiable update for it, and proceed to the next image for which there is **stored** image metadata, if any.  If the terminating value for this mapping is false, continue the search for verified image metadata for this image by continuing to the next mapping, discarding the image metadata for this image from the repositories other than the Director repository.
-7. The image metadata **stored** for each image is now verified.
+2. Refresh the top-level metadata for the Director repository, by following the steps in {{refresh_toplevel_metadata}}. The first repository listed in the repository mapping metadata is always the Director repository.
+3. Iterate through the image metadata listed in the Director repository's Targets metadata file, and **store** the image metadata this ECU is responsible for verifying. A primary ECU is responsible for verifying ALL image metadata; a secondary ECU is responsible for verifying only image metadata that includes its own ECU identifier and hardware ID.
```

Comment:
More succinct 👍

No longer requires that a Primary check to make sure that unknown ECUs aren't being given instructions, but that's probably better anyway.  That seems like an implementation choice for downstream.